workflows/chatbot/inference/generate.py

```diff
@@ -730,15 +724,16 @@ def generate_output():
         if output_word_len != 1
         else 0
     )
+
     if return_stats:
         stats = {
-            "input_token_len": input_token_len,
-            "output_word_len": output_word_len,
-            "duration": duration,
-            "first_word_latency": first_word_latency,
-            "msecond_per_word": msecond_per_word,
+            "input_token_len": str(input_token_len) + " ms",
```

Comment:
why token unit is ms?

Reply:
'input_token_len' / 'output_word_len' should not have units. Thanks, I'll correct it.